Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+flask = "*"
+
+[requires]
+python_version = "3.7"
```

Comment:
Please update to use Python 3.6. I could not get the code to run locally with 3.7, but 3.6.9 works just fine.,

Reply:
Hi Christian,
I have updated python version in pipfile. Please review again.